src/AppContext.php

```diff
@@ -3,11 +3,11 @@
 namespace WPGraphQL;
 
 use GraphQL\Error\UserError;
-use WPGraphQL\Data\Loader\CommentLoader;
-use WPGraphQL\Data\Loader\MenuItemLoader;
-use WPGraphQL\Data\Loader\PostObjectLoader;
-use WPGraphQL\Data\Loader\TermObjectLoader;
-use WPGraphQL\Data\Loader\UserLoader;
+use WPGraphQL\Data\Loader\Comment_Loader;
+use WPGraphQL\Data\Loader\Menu_Item_Loader;
+use WPGraphQL\Data\Loader\Post_Object_Loader;
+use WPGraphQL\Data\Loader\Term_Object_Loader;
+use WPGraphQL\Data\Loader\User_Loader;
```

Comment:
I don't think these changes are valuable to the plugin. We aren't currently considering merging the plugin into core, which is the only instance that I think these changes would make sense. These are going to cause breaking changes for people extending the plugin which will be a poor developer experience without much to gain.

Reply:
I hadn't actually gotten to the files in the `src` directory root yet. So I only made enough changes to make the files in the `connection` and `data` directory work.